degirum_tools/zone_count.py

```diff
@@ -37,6 +37,7 @@
 from .result_analyzer_base import ResultAnalyzerBase
 from .math_support import AnchorPoint, get_anchor_coordinates
 
+Mat = type[np.ndarray[Any, np.dtype[Any]]]

```

Comment:
Well, better not to do that - you override opencv very basic type

Reply:
Yes, was just testing. I didn't understand why it did not think a ndarray is matlike. because if you look at cv2.typing.matlike it was np.ndarray[any, np.dtype[generic]]. But it still threw an error